addons/website_event_track/models/event_track.py

```diff
@@ -273,8 +274,11 @@ def _compute_contact_phone(self):
     def _compute_end_date(self):
         for track in self:
             if track.date:
-                delta = timedelta(minutes=60 * track.duration)
-                track.date_end = track.date + delta
+                try:
+                    delta = timedelta(minutes=60 * track.duration)
+                    track.date_end = track.date + delta
+                except OverflowError:
+                    raise UserError(_("The entered duration creates a date too far into the future."))
```

Comment:
Would say it's better to have a constraint for this. So it fails even if this isn't computed (even though it will always be computed since it's stored, but that could change)

If we overflow here, it would be valid to just say `track.date_end = datetime.max` as that's what makes the most sense. No need to raise imo.